app/data_model/models.py

```diff
@@ -12,13 +12,17 @@ class QuestionnaireState(db.Model):
     __tablename__ = 'questionnaire_state'
     user_id = db.Column('userid', db.String, primary_key=True)
     state = db.Column('questionnaire_data', db.String)
+    version = db.Column('version', db.Integer)
+    last_updated = db.Column('last_updated', db.Integer)
+    created = db.Column('created', db.Integer)
```

Comment:
I think these column names have regressed in the merge of the database refactor, they should be `updated_at` and `created_at`